backticks -> tildes (#14434)
cockroachdb/docs: StyleGuide.md

#### Code block

> Use a code block to provide executable code samples. A code block has an opening and closing set of 3 tildes (`~~~`). A code block supports syntax highlighting if you add the language name immediately after the first line of tildes. There should be one returned line before and after a code block, for better Markdown readability. For example:

Author: ```suggestion
Use a code block to provide executable code samples. A code block has an opening and closing set of 3 tildes (`~~~`) or 3 backticks (` ``` `). A code block supports syntax highlighting if you add the language name immediately after the first line of tildes or backticks. There should be one returned line before and after a code block, for better Markdown readability. For example:
```

Reviewer: ```suggestion
Use a code block to provide executable code samples. A code block has an opening and closing set of 3 tildes (`~~~`) or 3 backticks (`` ``` ``). A code block supports syntax highlighting if you add the language name immediately after the first line of tildes. There should be one returned line before and after a code block, for better Markdown readability. For example:
```
